app/assets/stylesheets/hyku.scss

```diff
@@ -124,3 +124,86 @@ footer.navbar {
     padding: $nav-link-padding;
   }
 }
+
+.group-listing {
+  .panel-heading {
+    line-height: 2.4em;
+  }
+}
+
+.panel-tab-wrapper {
+  margin-top: 5em;
+  padding-bottom: 1px;
+}
+
+.nav-escape-link {
+  margin-top: -30px;
+}
+
+.group-listing,
+.group-user-listing {
+  .table td {
+    vertical-align: middle;
+  }
+}
+
+.group-listing .table {
+  margin-bottom: 0;
+  margin-top: 1em;
+  padding: 0 15px;
+
+  td:nth-child(2),
+  th:nth-child(2) {
+    text-align: right;
+    width: 5em;
+  }
+
+  td:nth-child(3),
+  th:nth-child(3) {
+    text-align: right;
+    width: 9em;
+  }
+
+  td:nth-child(4),
+  th:nth-child(4) {
+    text-align: center;
+    width: 13em;
+  }
+}
+
+.full-width-container {
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.list-scope {
+  position: relative;
+
+  .list-scope__query {
```

Comment:
This is a valid BEM-style class name. The SCSS lint rule for `SelectorFormat` has been updated to `strict_BEM` to allow this.